.editorconfig

```diff
@@ -0,0 +1,13 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
What we would like is to keep only the 2-trailing-whitespaces. Some editors do that but I don't remember which.
https://github.com/editorconfig/editorconfig/issues/391

Reply:
editorconfig removed